.github/workflows/tests_lqcpu_python.yml

```
@@ -79,16 +76,25 @@ jobs:
 
       - name: Get required Python packages
         run: |
-          cd main
           python -m pip install -r requirements-dev.txt
 
-      - name: Create device wheel
-        id: create_lq_wheel
+      # TODO: Update after Release 0.37
+      - name: Create device wheel ${{ inputs.lightning-version }}
+        if: inputs.lightning-version != 'stable'
         run: |
-          cd main
           PL_BACKEND=${{ matrix.pl_backend }} python scripts/configure_pyproject_toml.py
-          CMAKE_ARGS="-DENABLE_BLAS=${{ matrix.blas }} -DLQ_ENABLE_KERNEL_OMP=ON -DENABLE_PYTHON=ON -DCMAKE_CXX_COMPILER=$(which g++-$GCC_VERSION)" \
-          python -m build
+          CMAKE_ARGS="-DENABLE_BLAS=${{ matrix.blas }} -DLQ_ENABLE_KERNEL_OMP=ON -DENABLE_PYTHON=ON" python -m build
+          cd dist
+          WHEEL_NAME=$(ls *.whl)
```

Do we need quotations for the assignment?
```suggestion
          WHEEL_NAME="$(ls *.whl)"
```

---

I don't think so.